src/app/common/token-utils.ts

```diff
@@ -14,3 +17,30 @@ export function getFormattedAmount(amount: string) {
       }
     : { value: amount, isAbbreviated: false };
 }
+
+export function isIconUrl(iconString: string): boolean {
+  return isValidUrl(iconString);
+}
```

Comment:
this fn doesn't really add/do anything, could use `isValidUrl` directly, or if you think renaming is valuable, could be

```
const isIconUrl = isValidUrl
```